packages/rug/src/values.js

```diff
@@ -209,6 +208,10 @@ export const opacity = {
  * Shadows
  */
 
+const shadow = (color) => (
```

Comment:
Can we leave this in utils?

There doesn't seem to be a big difference in shadow vs shadows at this stage.